feedback/service.py

```diff
@@ -22,6 +22,30 @@ def send_notifications(issue: Issue, send_email_to_reporter: bool) -> None:
         t.start()
 
 
+def send_feedback_notification(
+    user_email: str | None, satisfaction_rating: str, how_can_we_improve: str
+) -> None:
+    if settings.NOTIFY_ENABLED:
+        client = get_notify_api_client()
+        personalisation = {
+            "userEmail": user_email or "",
+            "satisfactionRating": satisfaction_rating,
+            "how_can_we_improve": how_can_we_improve,
```

Comment:
Looking at the notify template for this, seems like it might be expecting the key to be `howCanWeImprove`

Reply:
Oops, that's a typo!